pkl-core/src/test/files/LanguageSnippetTests/input/modules/invalidOutput2.pkl

```diff
@@ -0,0 +1,3 @@
+class Test {}
+
+output: Test = new {}
```

Comment:
Move this and `invliadOutput1` to `input/errors`.

I know we have some other error tests in `input/modules`, but those should probably be moved too. The intention is to put all testing around error messages in the `errors/` dir.